tailwind.config.js

```diff
@@ -1,7 +1,84 @@
 export default {
   content: ["./index.html", "./src/**/*.{js,jsx,ts,tsx}"],
+  darkMode: "class",
   theme: {
-    extend: {},
+    extend: {
+      perspective: {
+        1000: "1000px",
+      },
+      rotate: {
+        "y-180": "rotateY(180deg)",
+      },
+      colors: {
+        primary: {
+          50: "#f0f9ff",
+          100: "#e0f2fe",
+          200: "#bae6fd",
+          300: "#7dd3fc",
+          400: "#38bdf8",
+          500: "#0ea5e9",
+          600: "#0284c7",
+          700: "#0369a1",
+          800: "#075985",
+          900: "#0c4a6e",
+        },
+        // Boje za različita stanja kartica
+        card: {
+          new: "#22c55e", // zelena za nove kartice
+          learning: "#f59e0b", // narančasta za kartice u učenju
+          review: "#3b82f6", // plava za kartice za ponavljanje
+        },
+      },
+
+      spacing: {
+        card: "1.25rem", // Standardni padding za kartice
+        "card-lg": "1.875rem", // Veći padding za veće kartice
+      },
+
+      // Prilagođene sjene koje će dati našim karticama 3D izgled
+      boxShadow: {
+        card: "0 2px 4px rgba(0,0,0,0.1), 0 8px 16px rgba(0,0,0,0.1)",
+        "card-hover": "0 4px 6px rgba(0,0,0,0.1), 0 12px 24px rgba(0,0,0,0.1)",
+      },
+
+      // Prilagođene animacije za okretanje kartica
+      keyframes: {
+        "card-flip": {
+          "0%, 100%": { transform: "rotateY(0deg)" },
+          "50%": { transform: "rotateY(180deg)" },
+        },
+      },
+      animation: {
+        flip: "card-flip 0.6s ease-in-out",
+      },
+
+      // Prilagođeni border radiusi
+      borderRadius: {
+        card: "1rem",
```

Comment:
please create pixel sizes for border radius values and then you can leave card and `card-lg` values. For example, we can have `2: 0.125rem`, `4: 0.25rem`..., all the way up to 20px and then `card: 1rem`...